Source/Scene/OIT.js

```diff
@@ -525,6 +527,9 @@ define([
         var framebuffer = passState.framebuffer;
         var length = commands.length;
 
+        var shadowsEnabled = scene.frameState.shadowHints.shadowsEnabled;
+        var scene2D = scene.mode === SceneMode.SCENE2D;
```

Comment:
You don't have to check for SCENE2D here - `scene.frameState.shadowHints.shadowsEnabled` will be false if the scene isn't 3D.